packages/frontend/package.json

```diff
@@ -20,9 +20,9 @@
     "test": "mocha",
     "typecheck:ci": "yarn typecheck",
     "typecheck": "tsc --noEmit",
-    "storybook": "start-storybook -p 6006 --no-open",
-    "storybook:build": "(cd ../.. && wsrun -p @l2beat/{shared,config} -recsm build --module es6) && build-storybook",
-    "storybook:screenshot": "yarn storycap --serverCmd \"npx http-server ./storybook-static -p 9001\" http://localhost:9001 -V 2560x1440 -V 390x844"
+    "storybook": "storybook dev -p 6006 --no-open",
+    "storybook:build": "(cd ../.. && wsrun -p @l2beat/{shared,config} -recsm build --module es6) && storybook build",
+    "storybook:screenshot": "yarn storycap --serverCmd \"npx http-server ./storybook-static -p 9001\" http://localhost:9001 --captureTimeout 50000 --parallel 8"
```

Comment:
50s is quite long. Have you tried shorter timeouts?

Reply:
Unfortunately, charts are taking so long to set up (~40000ms). I will revisit chart stories to try to optimize it but I tried once and could not do it :(